### PR TITLE
YJDH-153 | KS-Employer: Add Sentry logging

### PR DESCRIPTION
**Pull request by klemmari1**
## Description :sparkles:

- Add initial Sentry setup for the Employer UI.
- KS-Backend: Add support for a header used by the frontend sentry: "sentry-trace".

## Issues :bug:

[YJDH-153](https://helsinkisolutionoffice.atlassian.net/browse/YJDH-153)

### CODE REVIEW

**Review by haavikko — APPROVED** [review]
Looks good to me! Maybe if you have time, document something about SENTRY_AUTH_TOKEN configuration in different environments, like is it required and where to configure it

File: frontend/kesaseteli/employer/sentry.server.config.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
```
Comment (nenonja) [inline frontend/kesaseteli/employer/sentry.server.config.js:5]: Is this enough or should we wrap the server also with sentry? https://github.com/City-of-Helsinki/yjdh/blob/develop/frontend/shared/src/server/next-server.js#L27-L44

with this https://docs.sentry.io/platforms/javascript/guides/nextjs/:

```
import { withSentry } from "@sentry/nextjs";

const handler = async (req, res) => {
  res.status(200).json({ name: "John Doe" });
};

export default withSentry(handler);
```

**Review by nenonja — COMMENTED** [review]
Quickly read through and I didnt test this in anyway... 
One thing to remember is to check on staging and production envs that the sentry works on them too.

File: frontend/kesaseteli/employer/sentry.server.config.js (hunk 1 above)
Comment (nenonja) [reply frontend/kesaseteli/employer/sentry.server.config.js:5]: as we spoke, no need to do this

**Pull request merged**